### PR TITLE
refactor: normalize the fields naming in adaptor devices

### PR DESCRIPTION
**Pull request by thxCode**
**Fixes:**

https://github.com/cnrancher/octopus/issues/118

**Problem:**

In the CRD of the device, there are many non-standard fields that we need to adjust.

**Solution:**

- Clarify BLE adaptor.
- Clarify Modbus adaptor.
- Clarify OPC-UA adaptor.
- Remove deprecated parameters `Parameters` from limb-adaptor grpc.

**Test plan:**

None

### CODE REVIEW

File: adaptors/mqtt/pkg/physical/device.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ type Device interface {
 
 // NewDevice creates a Device.
 func NewDevice(log logr.Logger, meta metav1.ObjectMeta, toLimb MQTTDeviceLimbSyncer) Device {
-	log.Info("Created")
+	log.Info("Created ")
```
Comment (guangbochen) [inline adaptors/mqtt/pkg/physical/device.go:30]: remove empty space?

File: pkg/suctioncup/connection/connection.go [hunk 2]
```diff
@@ -82,26 +84,40 @@ func (c *connection) Send(model *metav1.TypeMeta, device []byte, references map[
...
 		References: references,
-	}); err == nil {
-		err = <-c.interruptError
+	}); err != nil {
+		return
 	}
 
-	return err
+	// TODO should we parameterize the sending timeout?
+	var timeout = time.NewTimer(5 * time.Minute)
```
Comment (guangbochen) [inline pkg/suctioncup/connection/connection.go:103]: is 5 minutes too long?

Comment (guangbochen) [inline pkg/suctioncup/connection/connection.go:103]: is 5 minutes too long?

File: adaptors/mqtt/pkg/physical/device.go (hunk 1 above)
Comment (thxCode) [reply adaptors/mqtt/pkg/physical/device.go:30]: This is to align the `shutdown` log content.

File: adaptors/ble/api/v1alpha1/bluetoothdevice_types.go [hunk 3]
```diff
@@ -5,26 +5,8 @@ import (
...
-	// Specifies the extension of device.
-	// +optional
-	Extension *DeviceExtensionSpec `json:"extension,omitempty"`
-}
-
-type Parameters struct {
+// BluetoothDeviceParameters defines the desired parameters of BluetoothDevice.
+type BluetoothDeviceParameters struct {
 	// Specifies default device sync interval
 	// +kubebuilder:default:15s
```
[the remark this answers is not in the document]
Comment (thxCode) [reply adaptors/ble/api/v1alpha1/bluetoothdevice_types.go:11]: Updated

File: adaptors/opcua/deploy/e2e/dl_opcua.yaml [hunk 4]
```diff
@@ -18,26 +18,26 @@ spec:
...
+          description: mock byte string. Default value is "test123"
           readOnly: false
           visitor:
-            nodeID: ns=3;s=Constant
-          value: "2.33"
-          dataType: float
\ No newline at end of file
+            nodeID: ns=1;s=myByteString
+          type: byteString
+          value: "newString"
```
Comment (guangbochen) [inline adaptors/opcua/deploy/e2e/dl_opcua.yaml:43]: resolve newline at the end of file

**Pull request merged**